RedHatInsights/insights-inventory-frontend
# feat(InventoryTable): RHINENG-9872 - Implement export download notification

This adds polling for exports and downloading. as a temporary solution till the export service itself can notify the users.

**How to test:**

1) Open the Inventory Systems page (with the export feature enabled)
1.1) Use the rhn-engineering-sgraessl account to not stress the backend to much and make sure the export/download works.
2) Click the export and choose a format.
3) Verify that a "pending notification" was triggered.
4) Verify that a "downloading export notification" was triggered and the desired export has been downloaded.
